llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -150,6 +150,11 @@ static cl::opt<bool> EnableVectorFCopySignExtendRound(
     cl::desc(
         "Enable merging extends and rounds into FCOPYSIGN on vector types"));
 
+static cl::opt<bool>
```

Comment:
These flags aren't really intended for end users, and are difficult to discover. Is there a threshold or something you can add instead?

Reply:
I know what you mean about these options being mostly for debug. However, this compile time issue is more of a special case. My fear with adding a threshold is that this is target indep code and it can negatively impact performance on a whole set of targets. To avoid this I would still prefer to keep the default as it is now and add an option that would add a threshold value. So, either way, I get a debug option like this one.

Reply:
I fear that fear of touching other targets results in an increasing set of single purpose knobs like this, the weight of which is quite high.